LiveSplit.SuperMetroid.asl

```diff
@@ -863,6 +863,382 @@ gameTime
 	return TimeSpan.FromSeconds(current.totalTime);
 }
 
+isLoading
```

Comment:
Looks like this section got copy-pasted several times at the end of the file.

Reply:
No idea how this happened. I guessing it's something to do with notepad++ on my tablet. Won't use that for edits anymore...